packages/analytics-v1.1/src/core/analytics.js

```diff
@@ -988,19 +996,23 @@ class Analytics {
         // new event processing after analytics initialized  but integrations not fetched from BE
         this.toBeProcessedByIntegrationArray.push([type, rudderElement]);
       } else {
-        // get intersection between config plane native enabled destinations
-        // (which were able to successfully load on the page) vs user supplied integrations
-        const successfulLoadedIntersectClientSuppliedIntegrations = findAllEnabledDestinations(
-          clientSuppliedIntegrations,
-          this.clientIntegrationObjects,
-        );
+        try {
+          // get intersection between config plane native enabled destinations
+          // (which were able to successfully load on the page) vs user supplied integrations
+          const successfulLoadedIntersectClientSuppliedIntegrations = findAllEnabledDestinations(
+            clientSuppliedIntegrations,
+            this.clientIntegrationObjects,
+          );
 
-        // try to first send to all integrations, if list populated from BE
-        this.processAndSendEventsToDeviceMode(
-          successfulLoadedIntersectClientSuppliedIntegrations,
-          rudderElement,
-          type,
-        );
+          // try to first send to all integrations, if list populated from BE
+          this.processAndSendEventsToDeviceMode(
+            successfulLoadedIntersectClientSuppliedIntegrations,
+            rudderElement,
+            type,
+          );
+        } catch (err) {
+          handleError(err, `processAndSendEventsToDeviceMode::`);
+        }
       }
 
       const clonedRudderElement = R.clone(rudderElement);
```

Comment:
The code blocks from Ln 994-1016 should be completely isolated with it's own try-catch block around it to prevent any damage to the cloud mode events delivery below.

Reply:
Addressed